.github/workflows/kernel-apply-patch.sh

```diff
@@ -36,7 +36,12 @@ done
 
 popd >/dev/null || exit
 
-URL="https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tag/?h=v${VERSION_NEW}"
+curl -sA 'Chrome' -L 'http://www.google.com/search?hl=en&q=site%3Alwn.net+linux+'"${VERSION_NEW}" -o search.html
+URL=$(grep -o 'https://lwn.net/Articles/[0-9]\+' search.html | head -n 1)
```

Comment:
The `set -euo pipefail` will make this line fail if grep doesn't find anything, and the script exits.
```suggestion
URL=$(grep -m 1 -o 'https://lwn.net/Articles/[0-9]\+' search.html || true)
```

Reply:
```
$ curl -sA "Chrome" -L 'http://www.google.com/search?hl=en&q=site%3Alwn.net+linux+5.15.6' -o search.html
$ grep -m 1 -o 'https://lwn.net/Articles/[0-9]\+' search.html 
https://lwn.net/Articles/877286
https://lwn.net/Articles/876861
https://lwn.net/Articles/877285
https://lwn.net/Articles/876976
https://lwn.net/Articles/484685
```

So, didn't work, because apparently everything is just one line. I'll look into making it more robust.

Reply:
Ok, updated the suggestion

Reply:
Heh, I went with parentheses instead of braces, but in the end applied your suggestion too. But surely braces have some weird syntax quirks.